linear_algebra/code/CMakeLists.txt

```diff
@@ -102,10 +93,18 @@ target_link_libraries(wg21_linear_algebra
         std::mdspan
 )
 
-if (BUILD_TESTING)
-    include(CTest)
-    add_library(wg21_linear_algebra::wg21_linear_algebra ALIAS wg21_linear_algebra)
+include(CTest)
+if (${BUILD_TESTING})
```

Comment:
`if (BUILD_TESTING)` tests if the variable exists, not the value of the variable.  `if (${BUILD_TESTING})` tests the value.